### PR TITLE
claims page

### PR DESCRIPTION
**Pull request by hobbitronics**
### added content to claims page
- card layout as per designers intent
- created Row and ClaimCards components
- added Row, ClaimCards and ClaimCard to index
- imported Row to home and claims
- added ClaimCards to claims page
- added state to example item

### CODE REVIEW

File: src/components/mdc/Row.svelte [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<script>
+export let cols = '' //number of colums can be up to 12
+</script>
+
+<div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-{cols} {$$props.class}">
```
Comment (forevermatt) [inline src/components/mdc/Row.svelte:5]: Does `mdc-layout-grid__cell--span-` work without a number? If not, you may want to set a good default value for the `export let cols = ...` above.

File: src/pages/claims.svelte [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<script>
+  import { ClaimCards, Row } from '../components/'
+  import { Page, Button } from '@silintl/ui-components'
+  
+  const exampleItems = [
+    {
+      name: "Saxophone",
+      accountable_person: "John Russel",
+      last_changed: "5 days",
+      sate: { icon: 'chat' }
```
Comment (forevermatt) [inline src/pages/claims.svelte:10]:
```suggestion
      state: { icon: 'chat' }
```

File: src/pages/claims.svelte [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      accountable_person: "Priscilla Russel",
+      last_changed: "5 days",
+      state: { icon: 'chat' }
+    },
+  ]
+  </script>
+  
+  <Page layout="grid">
+    <Row cols={'12'}>
+      <Button raised url="./claims/newClaim">New claim</Button>
```
Comment (forevermatt) [inline src/pages/claims.svelte:47]: Is this dot needed here?

Comment (hobbitronics) [reply src/pages/claims.svelte:47]: nope

File: src/components/mdc/Row.svelte (hunk 1 above)
Comment (hobbitronics) [reply src/components/mdc/Row.svelte:5]: I've set the default of cols to 12

**Review by forevermatt — APPROVED** [review]

**Pull request merged**